### PR TITLE
Login sys

### PR DESCRIPTION
**Pull request by Parul76**

### CODE REVIEW

File: mrscake/src/components/Reg.jsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import { Button, Card, CardFooter, CardBody, CardGroup, Col, Container, Form, 
+    Input, InputGroup, InputGroupAddon, InputGroupText, Row } from 'reactstrap';
+class Reg extends Component {
```
Comment (abdelilahAchir) [inline mrscake/src/components/Reg.jsx]: What do you mean whit Reg?

File: mrscake/src/App.js [hunk 2]
```diff
@@ -10,10 +12,8 @@ import ContactUs from './components/ContactUs';
 import ProductTable from './components/ProductPage';
 import NavBar from './components/NavBar';
 import FooterPage from './components/Footer';
-import './App.css';
```
Comment (abdelilahAchir) [inline mrscake/src/App.js:13]: why do you erase app.css?

Comment (Alina-dev-front) [reply mrscake/src/App.js:13]: Agree, this is strange, moreover you add new styles to this file

File: mrscake/src/components/Reg.jsx (hunk 1 above)
Comment (Parul76) [reply mrscake/src/components/Reg.jsx]: I will change from Reg to Registration in next commit.

File: mrscake/src/App.js (hunk 2 above)
Comment (Parul76) [reply mrscake/src/App.js:13]: i added thats why i deleted

Comment (Parul76) [reply mrscake/src/App.js:13]: I added two styles in app.cs and i will add app.cs in next commit, I was checking something without app.cs.

File: Mrs_Cake/Controllers/LoginController.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+namespace Mrs_Cake.Controllers
+{
+
+   
+    [Route("api/[controller]")]
+   
+    
+    public class LoginController
+    {
+        [RoutePrefix("Api/login")]
```
Comment (Alina-dev-front) [inline Mrs_Cake/Controllers/LoginController.cs]: Does "A" in "Api/login" works for url?

File: Mrs_Cake/Controllers/LoginController.cs [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        public class LoginController : ApiController
+        {
+            UsersEntities DB = new UsersEntities();
+            [Route("InsertUser")]
+            [HttpPost]
+            public object InsertUser(Register Reg)
+            {
+                try
+                {
+                    UserLogin EL = new UserLogin();
```
Comment (Alina-dev-front) [inline Mrs_Cake/Controllers/LoginController.cs]: same issue with naming. EL = noone understands. Name it "UserLogin" or so?

File: Mrs_Cake/Controllers/LoginController.cs [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+                    throw;
+                }
+                return new Response
+                { Status = "Error", Message = "Invalid Data." };
+            }
+            [Route("Login")]
+            [HttpPost]
+            public Response UserLogin(Login login)
+            {
+                var log = DB.EmployeeLogins.Where(x => x.Email.Equals(login.Email) &&
```
Comment (Alina-dev-front) [inline Mrs_Cake/Controllers/LoginController.cs]: Who are employees?

File: Mrs_Cake/LoginDetails/login.cs [hunk 6]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Mrs_Cake.LoginDetails
+{
+    public class login
```
Comment (Alina-dev-front) [inline Mrs_Cake/LoginDetails/login.cs]: Name class Login

File: mrscake/src/App.js (hunk 2 above)
Comment (Alina-dev-front) [inline mrscake/src/App.js:13]: We still need App.css i think

File: Mrs_Cake/Controllers/LoginController.cs [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+{
+
+   
+    [Route("api/[controller]")]
+   
+    
+    public class LoginController
+    {
+        [RoutePrefix("Api/login")]
+        public class LoginController : ApiController
```
Comment (Alina-dev-front) [inline Mrs_Cake/Controllers/LoginController.cs]: Do we have here class inside class? how does it work?

File: mrscake/src/App.js (hunk 2 above)
Comment (Parul76) [reply mrscake/src/App.js:13]: I added app.cs.. see below

**Review by Alina-dev-front — APPROVED** [review]

**Pull request merged**